app/views/posts/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<section class="user-index">
+  <h1>Create New Post</h1>
+
+  <%= form_with model: @post, url: user_posts_path(@user) do |f| %>
+    <p>
+      <%= f.label :title %></br>
+      <%= f.text_field :title %>
+    </p>
+
+    <p>
+      <%= f.label :text %></br>
+      <%= f.text_area :text %>
+    </p>
+
+    <p><%= f.submit %></p>
+  <% end %>
+</section>
```

Comment:
- [ ] 🤔 There should be a form in the views/comments/new.html.erb file which accepts input for creating comments. Kindly create one ✍🏻 , It should be similar to the one you created in the views/posts/new.html.erb file which accepts input for creating posts. 👍🏻